### PR TITLE
Add explanatory blog post for Neptune plugin

### PR DESCRIPTION
**Pull request by Cole-Greer**
Adds a new blog post which introduces the Neptune plugin and explains some of it's key features and future plans.

### CODE REVIEW

File: blog/2024-04-26-nodestream-neptune-support/index.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Nodestream Neptune Support
+
+The recent release of Nodestream 0.12 has introduced support for Amazon Neptune as the first step towards broader multi-database support. Nodestream provides a flexible tool to perform bulk ETL into Amazon Neptune Database and Amazon Neptune Analytics.
```
Comment (zprobst) [inline blog/2024-04-26-nodestream-neptune-support/index.md:3]: Maybe worth linking to the release announcement on the "Nodestream 0.12" text?

File: blog/2024-04-05-nodestream-0-12/index.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ Core Contributors to this feature include:
 ### Parquet Support
 
 Many customers have data stored in parquet format. 
-Paquet is a columnar storage format that is optimized for reading and writing large datasets.
+Parquet is a columnar storage format that is optimized for reading and writing large datasets.
```
Comment (zprobst) [inline blog/2024-04-05-nodestream-0-12/index.md:67]: Thanks for catching these...

File: blog/2024-04-26-nodestream-neptune-support/index.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+## Overview
+
+Support for AWS Neptune is split into two modes, DB and Analytics. Both modes leverage the AWS SDK to load data via batched openCypher queries. Nodestream is compatible with Neptune DB engine version 1.2.1.1 or higher, as well as Neptune Analytics.
+
+## Capabilities
+
+Nodestream with Neptune currently supports standard ETL pipelines as well as time to live (TTL) pipelines. ETL pipelines enable bulk data ingestion into Neptune from a much broader range of data sources and formats than have previously been possible in Neptune.
+
+Nodestream's TTL mechanism also enables new capabilities not previously available in Neptune. By annotating ingested graph elements with timestamps, Nodestream is able to create pipelines which automatically expire and remove data that has passed a configured lifespan.
```
Comment (zprobst) [inline blog/2024-04-26-nodestream-neptune-support/index.md:15]: Is it a good idea to link the TTL docs? 
https://nodestream-proj.github.io/docs/docs/tutorials-intermediate/removing-data/

Comment (Cole-Greer) [reply blog/2024-04-26-nodestream-neptune-support/index.md]: Agreed, I added both suggested links.

**Pull request merged**